pipelines/rj_smas__disparo_pic_lembrete/flow.py

```
@@ -1,35 +1,43 @@
 # -*- coding: utf-8 -*-
-
+# flake8: noqa:E501
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)